### DISCUSSION TITLE
Empty home directory - easiest (best?) way to change the "No data available in table" message?

### DISCUSSION THREAD

**Discussion by redherring917b** [prasathmani/tinyfilemanager#855] [Q&A]
Hi there.  I'm wondering what the easiest (best?) way is to change the "No data available in table" message received when the user logs into an "empty" home directory.

Thanks!

**Comment by prasathmani** [comment]
add custom empty table message in the DataTable.
https://stackoverflow.com/a/31983571/1164642
```js
$('#main-table').DataTable({paging: false, info: false, order: [], columnDefs: [{targets: _targets, orderable: false}, {type: emptyType, targets: '_all',"oLanguage": {"sEmptyTable": "My Custom Message On Empty Table"}},]
```

**Reply by redherring917b** [reply]
Hi @prasathmani.  Thanks for your reply.

Do you mean replacing this existing line found in the $(document).ready( function () of tinyfilemanager.php:

`mainTable = $('#main-table').DataTable({paging: false, info: false, order: [], columnDefs: [{targets: _targets, orderable: false}, {type: emptyType, targets: '_all',},]});_`

with this revision:

`mainTable = $('#main-table').DataTable({paging: false, info: false, order: [], columnDefs: [{targets: _targets, orderable: false}, {type: emptyType, targets: '_all',"oLanguage": {"sEmptyTable": "My Custom Message On Empty Table"}},]});`

That seems right, and yet I tried that and it didn't seem to work?

Thanks again.
